conf/global/sql_connection.conf

```diff
@@ -50,3 +50,6 @@ sql_connection: {
 	db_database: "ragnarok"
 	//codepage:""
 }
+
+// Your custom config goes here.
+import: "conf/import/sql_connection.conf"
```

Comment:
I won't recommend this,
Just see how map/char/login server include sql_connection, and follow same structure in their imports.

Reply:
Should I update the comment to make it more clear? In my case I copy the same settings to 3 files that doesnt feel right

Reply:
I don't believe we look for an `import` key in this context - if we were do to this, the source would need to be modified accordingly, or am I missing something?